packages/lab/src/pill-next/PillNextCloseButton.tsx

```diff
@@ -0,0 +1,59 @@
+import { forwardRef, KeyboardEvent, ComponentPropsWithoutRef } from "react";
+import clsx from "clsx";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { makePrefixer } from "@salt-ds/core";
+import pillCloseButtonCss from "./PillNextCloseButton.css";
+import { CloseSmallIcon } from "@salt-ds/icons";
+
+export interface PillNextCloseButtonProps
+  extends ComponentPropsWithoutRef<"button"> {
+  /* If true the close button will be disabled */
+  disabled?: boolean;
+  onClose: () => void;
+}
+
+const withButtonBaseName = makePrefixer("saltPillNextCloseButton");
+
+export const PillNextCloseButton = forwardRef<
+  HTMLButtonElement,
+  PillNextCloseButtonProps
+>(function PillNextCloseButton({ disabled, onClose, ...rest }, ref) {
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-pill-next-close-button",
+    css: pillCloseButtonCss,
+    window: targetWindow,
+  });
+  const handleKeyUp = (e: KeyboardEvent) => {
```

Comment:
no check for `disabled` here but there is in `handleClose`.  Should this call `handleClose` and not `onClose`?